Makefile.am

```diff
@@ -149,9 +149,13 @@ venv:
 tests: tox
 	tox -e "tests,lib,network,virt,gluster,hooks"
 
-.PHONY: tests-storage
-tests-storage: tox
-	tox -e "storage"
```

Comment:
This break the CI in case we revert the next patch. Please move this change to the next commit. Every commit must be correct.